specs/slider/README.md

```diff
@@ -106,13 +119,13 @@ Comments to example 2
 
 ### Subcomponents (pseudo elements)
 
-| selector   | description                              | type                                     |
-| ---------- | ---------------------------------------- | ---------------------------------------- |
-| ::handle   | Allows you to style the handle of the slider | HTML Element. This subcomponent has no subcomponents of its own* |
-| ::bar      | Allows you to style the bar of the slider | HTML Element. This subcomponent has no subcomponents of its own* |
-| ::progress | Allows you to style the progress part of the bar | HTML Element. This subcomponent has no subcomponents of its own* |
+| selector   | description                              |
+| ---------- | ---------------------------------------- |
+| ::handle   | Allows you to style the handle of the slider |
+| ::bar      | Allows you to style the bar of the slider |
+| ::progress | Allows you to style the progress bar of the slider |
+
```

Comment:
Good catch. Actually, I made it intentionally. After thinking about marks, I came to the conclusion that we will not give the ability to style them (unless we have a feature request / real need). 
I want to keep basic components as simple as possible. 

If you look at the design (https://zpl.io/VDDKWWV) you will notice that marks that marks have the same style as ::progress & ::bar pseudo-elements.